types/ember__object/v3/index.d.ts

```diff
@@ -163,6 +163,8 @@ export function get<T, K extends keyof T>(
     obj: T,
     key: K
 ): UnwrapComputedPropertyGetter<T[K]>;
+export function get(obj: unknown, key: string): unknown;
```

Comment:
Since we're backporting to v3, would be good to also copy `test/get-set.ts` to `v3/test/` (and add to `v3/tsconfig.json`).